diff --git a/Montage/Makefile.LINUX b/Montage/Makefile.LINUX
index 57ea4eb..02cf1e3 100644
--- a/Montage/Makefile.LINUX
+++ b/Montage/Makefile.LINUX
@@ -53,14 +53,14 @@ LIBS   =	-L../lib -lwcs -lcfitsio -lcoord -lmtbl -lsvc \
 		   -lwww -lboundaries -lpixbounds -ltwoplane -lnsl -lm
 
 SBINS  =  mConvert mFlattenExec mHdrCheck mHdrtbl mTblSort mTileHdr mTileImage
-MBINS  =  mProjExecMPI mFitExecMPI mDiffExecMPI mBgExecMPI mAddMPI  mAddExecMPI
+MBINS  =  mProjExecMPI mFitExecMPI mDiffExecMPI mAddMPI  mAddExecMPI
 
 BINS   = 	$(SBINS)
 
 
 # uncomment the next two lines to build MPI modules
-# MPICC  =	mpicc
-# BINS = 		$(SBINS) $(MBINS)
+MPICC  =	mpicc
+BINS = 		$(SBINS) $(MBINS)
 
 
 .c.o:
diff --git a/Montage/checkHdr.c b/Montage/checkHdr.c
index 15d3b68..40da192 100644
--- a/Montage/checkHdr.c
+++ b/Montage/checkHdr.c
@@ -75,7 +75,7 @@ char ctype1[1024];
 char ctype2[1024];
 
 char *hdrCheck_outfile = (char *)NULL;
-FILE *fout;
+FILE *fout_1;
 
 int CHdebug    = 0;
 int errorCount = 0;
@@ -185,9 +185,9 @@ int checkHdr(char *infile, int hdrflag, int hdu)
 
    if(hdrCheck_outfile)
    {
-      fout = fopen(hdrCheck_outfile, "w+");
+      fout_1 = fopen(hdrCheck_outfile, "w+");
 
-      if(fout == (FILE *)NULL)
+      if(fout_1 == (FILE *)NULL)
       {
          fprintf(fstatus, "[struct stat=\"ERROR\", msg=\"Cannot open output file %s.\"]\n", hdrCheck_outfile);
          fflush(fstatus);
@@ -759,8 +759,8 @@ int errorOutput(char *msg)
 {
    if(hdrCheck_outfile)
    {
-      fprintf(fout, "%s\n", msg);
-      fflush(fout);
+      fprintf(fout_1, "%s\n", msg);
+      fflush(fout_1);
 
       ++errorCount;
    }
diff --git a/Montage/get_hfiles.c b/Montage/get_hfiles.c
index c26b2a7..1faf104 100644
--- a/Montage/get_hfiles.c
+++ b/Montage/get_hfiles.c
@@ -31,7 +31,7 @@ extern int   hdrlen;
 int  get_hhdr   (char*, struct Hdr_rec*, char*);
 void print_hrec (struct Hdr_rec*);
 
-struct Hdr_rec hdr_rec;
+struct Hdr_rec hdr_rec_1;
 
 
 /* Recursively finds all FITS files     */
@@ -70,13 +70,13 @@ void get_hfiles (char *pathname)
       sprintf (dirname, "%s/%s", pathname, entry->d_name);
 
       if(strncmp(dirname, "./", 2) == 0)
-	 strcpy (hdr_rec.fname, dirname+2);
+	 strcpy (hdr_rec_1.fname, dirname+2);
       else
-	 strcpy (hdr_rec.fname, dirname+hdrlen);
+	 strcpy (hdr_rec_1.fname, dirname+hdrlen);
 
       if(debug)
       {
-	 printf("DEBUG: [%s] -> [%s]\n", dirname, hdr_rec.fname);
+	 printf("DEBUG: [%s] -> [%s]\n", dirname, hdr_rec_1.fname);
 	 fflush(stdout);
       }
 
@@ -112,7 +112,7 @@ void get_hfiles (char *pathname)
 	    { 
 	       msg[0] = '\0';
 
-	       istatus = get_hhdr (dirname, &hdr_rec, msg);
+	       istatus = get_hhdr (dirname, &hdr_rec_1, msg);
 
 	       if (istatus != 0) 
 		  failed += istatus;
diff --git a/Montage/montage.h b/Montage/montage.h
index 7912116..a005820 100644
--- a/Montage/montage.h
+++ b/Montage/montage.h
@@ -1,4 +1,4 @@
 #ifndef MONTAGE
-FILE *fstatus;
 #define MONTAGE
+static FILE *fstatus;
 #endif
diff --git a/lib/src/coord/coord.h b/lib/src/coord/coord.h
index 7a9eda9..65428e8 100644
--- a/lib/src/coord/coord.h
+++ b/lib/src/coord/coord.h
@@ -1,7 +1,7 @@
 #ifndef ISIS_COORD_LIB
 
 
-int coord_debug;
+static int coord_debug;
 
 
 struct COORD                /* Definition of coordinate structure            */
